### PR TITLE
Change domain.ErrLogger to proxy to stderr and Rollbar

### PR DESCRIPTION
**Pull request by fillup**

### CODE REVIEW

File: application/domain/domain.go [hunk 1]
```diff
@@ -224,6 +224,34 @@ type AppError struct {
...
+// ErrLogProxy wraps standard error logger plus sends to Rollbar
+type ErrLogProxy struct {
+	LocalLog  log.Logger
+	RemoteLog *rollbar.Client
+}
+
+func (e *ErrLogProxy) SetOutput(w io.Writer) {
+	e.LocalLog.SetOutput(w)
+}
+func (e *ErrLogProxy) Printf(format string, a ...interface{}) {
```
Comment (Baggerone) [inline application/domain/domain.go]: Maybe, add blank line before functions.

File: application/listeners/posthelpers.go [hunk 2]
```diff
@@ -260,7 +260,7 @@ func sendNotificationRequestFromCompletedToAcceptedOrDelivered(params senderPara
 }
 
 func sendNotificationEmpty(params senderParams) {
-	domain.ErrLogger.Print("Notification not implemented yet for " + params.template)
+	domain.ErrLogger.Printf("Notification not implemented yet for %s", params.template)
```
Comment (Baggerone) [inline application/listeners/posthelpers.go]: Minor note - you could have left this the same other than adding the `f` to `Print`.  (Same for some of the others.)

Comment (Schparky) [reply application/listeners/posthelpers.go]: Could they have been left alone if a `Print` is defined in `ErrLogProxy`?

File: application/domain/domain.go (hunk 1 above)
Comment (fillup) [reply application/domain/domain.go]: k

**Pull request merged**